contracts/mocks/account/AccountERC7702WithModulesMock.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.24;
+
+import {PackedUserOperation} from "@openzeppelin/contracts/interfaces/draft-IERC4337.sol";
+import {AccountCore} from "../../account/AccountCore.sol";
+import {Account} from "../../account/Account.sol";
+import {AccountERC7579} from "../../account/extensions/AccountERC7579.sol";
+import {ERC7821} from "../../account/extensions/ERC7821.sol";
+import {AbstractSigner} from "../../utils/cryptography/AbstractSigner.sol";
+import {SignerERC7702} from "../../utils/cryptography/SignerERC7702.sol";
+
+abstract contract AccountERC7702WithModulesMock is Account, AccountERC7579, SignerERC7702 {
+    function execute(
+        bytes32 mode,
+        bytes calldata executionData
+    ) public payable virtual override(AccountERC7579, ERC7821) {
+        // Force resolution to AccountERC7579 (discard ERC7821 implementation)
+        AccountERC7579.execute(mode, executionData);
+    }
+
+    function supportsExecutionMode(
+        bytes32 mode
+    ) public view virtual override(AccountERC7579, ERC7821) returns (bool result) {
+        // Force resolution to AccountERC7579 (discard ERC7821 implementation)
+        return AccountERC7579.supportsExecutionMode(mode);
+    }
```

Comment:
Thinking more about this. I'm leaning towards removing ERC-7821 from Account by default. I envision the Wizard starts with Account.sol and requires the user to select an execution mechanism:

- [ ] Minimal (7821)
- [ ] Modular (7579)